test/e2e/test_guest_user_reads.py

```diff
@@ -91,20 +135,49 @@ def test_guest_user_private_model_read(
         assert "not found" in str(error)
 
 
+def test_guest_user_private_model_logged_data_read(
+    client: LayerClient,
+    populated_project: Project,
+    guest_context,
+    guest_client: LayerClient,
```

Comment:
i understand why you have separate tests, but they do create separate runs every time which is not great for our costs and ci/test durations

can we simply have 2 tests?
1. check all private things cannot be access from guest
2. check all public things can be accessed from guest

also serves as more straight to the point documentation

Reply:
yes, sorry i should have spotted this earlier